README.md

```diff
@@ -15,6 +15,35 @@ The following examples use the available authentication methods to authenticate
 The keys in the `env` secret are exposed in the `checkout` and `command` as environment variables. The git-credentials are exposed as an environment variable `GIT_CONFIG_PARAMETERS` and are also exposed in the `checkout` and `command`.
 
 ### AppRole Authentication
+You can read more about Vault's AppRole auth method in their [documentation](https://developer.hashicorp.com/vault/docs/auth/approle).
```

Comment:
I think this still needs a `why`; we're leaving it up to the reader to infer everything about the secret ID, but we should include some reasoning for it at the beginning of the section:

Why is `VAULT_SECRET_ID` relevant to the plugin? The link to the docs can be "additional reading"